### PR TITLE
Modular migration

### PR DESCRIPTION
**Pull request by askchrisn**
**User-Facing Changes**

**Description**

### CODE REVIEW

**Comment by mir-lca** [comment]
I'm trying to build this branch as a docker image and getting into some issues, examples:

> #12 72.50   ERROR in ../../packages/studio-base/src/mir/Markers.ts 5:0-28
#12 72.50   Module not found: Error: Can't resolve 'earcut' in '/src/packages/studio-base/src/mir'

> #12 72.50   ERROR in ../../packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts 12:0-25
#12 72.50   Module not found: Error: Can't resolve 'png-ts' in '/src/packages/studio-base/src/panels/ThreeDeeRender/renderables'

> #12 72.50   ERROR in ./packages/studio-base/src/mir/Markers.ts:2:30
#12 72.50   TS6133: 'MIR_NAVIGATION_MAP_DATATYPES' is declared but its value is never read.

**Comment by mir-lca** [comment]
I have a fix for the docker image build issue. Can you give me write access to this repo?

File: packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts
Comment (anthiras) [inline packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts]: There's a lot of code commented out in this file, is this intentional?

Comment (askchrisn) [reply packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts]: Yeah this was all apart of the original MiR edited code.
I can remove it but was thinking we might want it. Your call

Comment (anthiras) [reply packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts]: Sounds like we need an OccupancyGrids in the mir folder then, if there are so many changes to it.

Comment (mir-lca) [reply packages/studio-base/src/panels/ThreeDeeRender/renderables/OccupancyGrids.ts]: The modified files outside the mir folder were breaking the docker build and I thought they were leftovers from the migration so I reset them. Sounds like it was a mistake.

Are we keeping any modifications outside of the mir folder?

**Review by anthiras — APPROVED** [review]
Great, there are very few changes to existing files now.

**Pull request merged**
